company/values.md

```diff
@@ -64,3 +64,19 @@ We document our goals and plans publicly so we can engage with customers to buil
 > I am working on improving the security of our product, but I feel uncomfortable clearly documenting what I am doing (e.g., in public status updates, issues, PR descriptions, the changelog) because I am worried that it will make Sourcegraph look bad.
 
 Our code and PRs are public, so lack of transparency is only obfuscation, not security. Unless something is immediately exploitable and would put customer data at risk, you should be completely transparent about it, just like any other non-security improvement.
+
+## High Agency
+
+Every person at Sourcegraph has both the power and the responsibility to help make Sourcegraph a better place and a better product.
+
+> “Life can be much broader once you discover one simple fact: Everything around you that you call life was made up by people that were no smarter than you and you can change it, you can influence it, you can build your own things that other people can use. Once you learn that, you'll never be the same again.” - Steve Jobs
+
+- As a [handbook-first](https://about.sourcegraph.com/handbook/usage#why-handbook-first) company, all of us are free to discuss and implement changes to any aspect of our business.
```

Comment:
This bullet list feels arbitrary. Why is it in bullets and the rest isn't?

Reply:
Agreed - was considering having a longer list of examples, but in current state, bullets don't make sense.